src/patterns/ec2-app/base.test.ts

```diff
@@ -1072,4 +1072,27 @@ describe("the GuEC2App pattern", function () {
       LoadBalancerAttributes: Match.arrayWith([Match.objectLike({ Key: "access_logs.s3.prefix", Value: "test-2" })]),
     });
   });
+
+  it("uses the latest security policy", function () {
+    const stack = simpleGuStackForTesting();
+    new GuEc2App(stack, {
+      applicationPort: 3000,
+      app: "test-gu-ec2-app",
+      access: { scope: AccessScope.PUBLIC },
+      instanceType: InstanceType.of(InstanceClass.T4G, InstanceSize.MEDIUM),
+      monitoringConfiguration: { noMonitoring: true },
+      userData: "#!/bin/dev foobarbaz",
+      certificateProps: {
+        domainName: "domain-name-for-your-application.example",
+      },
+      scaling: {
+        minimumInstances: 1,
+      },
+      instanceMetadataHopLimit: 2,
+    });
+
+    Template.fromStack(stack).hasResourceProperties("AWS::ElasticLoadBalancingV2::Listener", {
+      SslPolicy: "ELBSecurityPolicy-TLS13-1-2-2021-06",
```

Comment:
I have specifically hardcoded the string here instead of using the enum variant `SslPolicy.RECOMMENDED_TLS`, as AWS might change the underlying string, so this test will help us catch unexpected changes.